web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -151,24 +151,27 @@ const HfiCalculatorPage: React.FunctionComponent = () => {
   }
 
   const copyTable = () => {
-    if (isWeeklyView) {
+    if (isUndefined(selectedFireCentre)) {
```

Comment:
To simplify, how about:

```
if(!isUndefined(selectedFireCentre)) {
// do work
}
```

Removes the else and unnecessary state change to the copy state